wdl-lint/src/rules/import_sort.rs

```diff
@@ -25,7 +25,7 @@ fn import_not_sorted(span: Span) -> Diagnostic {
     Diagnostic::note("imports are not sorted lexicographically")
         .with_rule(ID)
         .with_highlight(span)
-        .with_fix("sort the imports lexicographically")
+        .with_fix("sort the imports lexicographically") // TODO: Provide the correct sorting
```

Comment:
Rather than a `TODO`, should this be an open issue? I think we should provide `fix` output that contains the corrected code when possible.

Reply:
Sure I'll open one now. And mark it `good first issue` as well.

I think it's fine to leave the `TODO` comment here though, unless you think it should be deleted?

Reply:
No, I think that's fine to have both.